skills/nodej/bulk-update-skills.js

```diff
@@ -0,0 +1,40 @@
+const execSync = require('child_process').execSync;
+
+let removeSkills = ['Juggling','Fire Breathing','Origami'];
+
+if(process.platform === 'linux'){
```

Comment:
Could this work on macOs also?
It could probably also work on Windows if the script detects the OS and changes `gc` to `gc.exe` and uses `\` instead of `/` for path separators if it's Windows.

Reply:
I'm not sure if this will work on macOS as I do not have a machine to test it to.. I've pushed a change to address this comment.